recipes/PyYAML-ft/build.sh

```diff
@@ -0,0 +1,9 @@
+export PYYAML_FORCE_CYTHON=1
+export PYYAML_FORCE_LIBYAML=1
+export CFLAGS="-I%LIBRARY_INC%"
+export LDFLAGS="-L%LIBRARY_LIB%"
```

Comment:
This looks wrong. Do you need to set these variables at all?

If you are sure it is needed, shouldn't it be something like
```suggestion
export CFLAGS="-I${LIBRARY_INC}"
export LDFLAGS="-L${LIBRARY_LIB}"
```

Generally speaking, the conda-forge scripts are quite good at setting these variables themselves, though I can unfortunately not access the logs at the moment.

Reply:
I'd verified that Windows and Linux were failing without the env vars set. It does seem odd that it worked on Linux since this indeed is a typo (not sure if Linux does env var substitution with `%`, it probably doesn't?).

Upon closer inspection just now, `CFLAGS` and `LDFLAGS` appear to be set the same way as in your local build. Not sure how macOS can succeed and Linux fail like this.

I'd propose to add them back for now and I can take a closer look after we've merged this. Does this seem fine to you?

Reply:
Seems like without canceling the flags, [the extension module gets the wrong tag](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1245585&view=logs&j=4cabe686-70ae-553a-7fd0-310379f2cbac&t=6a4fc7c9-c31a-5115-eff9-6479d72b69ff&l=1039). I'm not sure why that is, but I am hesitant to remove all of the standard flags.

@isuruf, would you have some guidance?